docs/src/main/asciidoc/micrometer.adoc

```diff
@@ -402,9 +402,9 @@ to inject any configuration attributes you need to configure the registry. Most
 like `quarkus-micrometer-registry-statsd`, define a producer for registry-specific configuration objects
 that are integrated with the Quarkus configuration model.
 
-== Support for the MicroProfile Metrics API
```

Comment:
These changes should be reverted.

This section is specifically about enabling the use of MP Metrics APIs with Micrometer, nothing to do with SmallRye Metrics.

cc @ebullient